pkg/domain/entities/images.go

```diff
@@ -244,6 +244,9 @@ type ImagePushOptions struct {
 	// integers in the slice represent 0-indexed layer indices, with support for negative
 	// indexing. i.e. 0 is the first layer, -1 is the last (top-most) layer.
 	OciEncryptLayers *[]int
+	// Add existing instances with requested compression algorithms to manifest list
```

Comment:
maybe 
```suggestion
	// If necessary, add clones of existing instances with requested compression algorithms to manifest list
```
?

OTOH as is, this matches the c/common comment

Reply:
done.